operator/cmd/mesh/install.go

```diff
@@ -145,9 +140,8 @@ func InstallCmd(ctx cli.Context) *cobra.Command {
 
 func Install(kubeClient kube.CLIClient, rootArgs *RootArgs, iArgs *InstallArgs, stdOut io.Writer, l clog.Logger, p Printer,
 ) error {
-	kubeClient, client, err := KubernetesClients(kubeClient, l)
-	if err != nil {
-		return err
+	if err := k8sversion.IsK8VersionSupported(kubeClient, l); err != nil {
```

Comment:
(not really a review comment):

_Ultimately_ I'd prefer to drop this - Helm charts [already support declaring a `minKubeVersion` in the manifest](https://helm.sh/docs/topics/charts/#the-chartyaml-file) and it would be better to respect that if present, versus use the hardcoded lower bound we have in `"istio.io/istio/istioctl/pkg/install/k8sversion"`

Reply:
Where do you see minkubeversion? I don't see it in the link or where in Helms code?

Though you can get the kube version from `.Capabilities.KubeVersion`. One annoyance is with `helm template` they will just make it up...

Reply:
I just mean `.Capabilities.KubeVersion` - it's a semver range so the min version is implicit.

The value is set in the chart and templating doesn't change it - `helm template` will however generate a "fake" kube version that `.Capabilities.KubeVersion` can be tested against at template time, since there is no real cluster - there's an arg if we you want to use a different "fake" version for templating purposes.

That "fake" version generated during templating shouldn't be relevant for our usages tho - k8is version check is only an install-time check for us, just like it is for Helm.

pretty much this would just be making sure the version the kube client reports falls in the chart-level `.Capabilities.KubeVersion` semver range for each chart we install, rather than relying on a hardcoded const Go like we do now.